tests/smoke/hello_test.go

```diff
@@ -27,10 +27,15 @@ func TestHelloBundle(t *testing.T) {
 
 	test.RequirePorter("install", "--reference", ref)
 	test.RequirePorter("installation", "show", "mybuns")
+	test.RequirePorter("installation", "logs", "show", "-i=mybuns")
 
 	test.RequirePorter("upgrade", "mybuns")
 	test.RequirePorter("installation", "show", "mybuns")
 
-	test.RequirePorter("uninstall", "mybuns")
+	// Check that we can disable logs from persisting
+	test.RequirePorter("uninstall", "mybuns", "--persist-logs=false")
```

Comment:
`--no-logs` instead here, right?

Reply:
Oops yeah. I just pushed a fix for the smoke tests. I had originally gone with --persist-logs but that wasn't a good flag name because in general boolean flags are easiest to use when they default to false and you only specify it when you want it to be true.